src/viam/sdk/spatialmath/orientation.cpp

```diff
@@ -145,6 +145,5 @@ proto::Orientation OrientationConfig::to_proto() const {
         }
     }
 }
-
```

Comment:
(minor) can we add this newline back? Just for consistency with the rest of the SDK :)